src/widget/wtrackmenu.cpp

```diff
@@ -236,6 +256,16 @@ void WTrackMenu::createMenus() {
                     }
                     m_pSearchRelatedMenu->setEnabled(
                             !m_pSearchRelatedMenu->isEmpty());
+                    if (!m_pSearchRelatedMenu->isEmpty()) {
+                        // We're interested in keypress Qt::Key_Left, so use our
+                        // event filter like we do for the crate checkboxes.
+                        for (const auto ch : m_pSearchRelatedMenu->children()) {
```

Comment:
Clang tidy here

```suggestion
                        for (auto *const ch : m_pSearchRelatedMenu->children()) {
```

Reply:
Ah, thanks! overlooked that.
Though, why is it suggesting
`auto *const ch` and not
`const auto* ch`?